helm-charts/common/retriever-usvc/values.yaml

```diff
@@ -69,7 +69,7 @@ podSecurityContext: {}
 
 securityContext:
   # NOTE: many dependent python modules(e.g. numpy, mpl, haystack, etc.) need to write to local disk
```

Comment:
Should the comment be adjusted now that the FS is r/o?

Reply:
done